source/features/tags-dropdown.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'dom-chef';
+import * as pageDetect from 'github-url-detection';
+import delegate, {DelegateEvent} from 'delegate-it';
+
+import * as api from '../github-helpers/api';
+import features from '../feature-manager';
+import {buildRepoURL} from '../github-helpers';
+import observe from '../helpers/selector-observer';
+
+let latestTags: string[] | undefined;
+
+const gql = `
+	repository() {
+		refs(refPrefix: "refs/tags/", last: 100) {
+			nodes {
+				name
+			}
+		}
+	}
+`;
+
+// `datalist` selections don't have an `inputType`
+function selectionHandler(event: DelegateEvent<Event, HTMLInputElement>): void {
+	const field = event.delegateTarget;
+	const selectedTag = field.value;
+	if (!('inputType' in event) && latestTags!.includes(selectedTag)) {
+		location.href = buildRepoURL('releases/tag', selectedTag);
```

Comment:
As expected, it's broken on unescaped tag names:

https://github.com/refined-github/refined-github/releases

<img width="384" alt="Screenshot 5" src="https://user-images.githubusercontent.com/1402241/231680677-b6de645c-de8e-4a73-bfed-fbc81ebe9427.png">

Reply:
All good:

- https://github.com/refined-github/sandbox/releases/tag/JavaScript%25250Aalert.(1)''%22%22%60%60%2526apos%3B)%3C!--%3E%3CTitle%3CStyle%3CScript%3CtextArea%3CiFrame%3CnoScript%3E%3Ck%3CKcontentEditableautoFocusOnFocus%3D%24%7B%3B%7B(alert)(1)%7D%3E%3CBaseHref%3DX55.is%3E--%3E

Also cool UI:

- https://github.com/refined-github/sandbox/pull/40


<img width="1263" alt="Screenshot 7" src="https://user-images.githubusercontent.com/1402241/231684346-b6eee072-80d7-4822-b3f1-d55b38f56dfb.png">